Manifest.toml

```diff
@@ -0,0 +1,929 @@
+# This file is machine-generated - editing it directly is not advised
+
+julia_version = "1.7.2"
```

Comment:
Why did the Manifest for ClimaCoupler change with this PR?

Reply:
Ooh! Good catch! I think the formatting did this.